draft-wang-ppm-dap-taskprov.md

```diff
@@ -296,6 +296,27 @@ A protocol participant MUST opt out if the task has expired.
 The behavior of each protocol participant is determined by whether or not they
 opt in to a task.
 
+## Supporting a Global HPKE Configuration {#global-hpke-config}
+
+In DAP, Clients need to know the HPKE configuration of each Aggregator before
+sending reports. (See HPKE Configuration Request in
+{{!DAP=I-D.draft-ietf-ppm-dap-02}}.) However, in a DAP deployment that supports
+`task_prov` extension, if a Client queries an Aggregator's `/hpke_config`
+endpoint with a task ID computed as described in {{construct-task-id}}, the
+task ID may not be configured in the Aggregator yet, because the Aggregator is
+still waiting for the first Client report with the `task_prov` extension
+to arrive.
+
+To mitigate this issue, if an Aggregator wants to support `task_prov` extension,
+it SHOULD support a global HPKE configuration independent of the task ID, for all
+tasks that are configured with `task_prov` extension. It MAY continue to support
+per-task HPKE configurations for other tasks that are configured out-of-band.
+
+In addition, if a Client wants to include `task_prov` extension in its report,
+it SHOULD NOT specify the `task_id` parameter when querying an Aggregator's
+`/hpke_config` endpoint, because the task ID may not have been configured
+in the Aggregator yet.
```

Comment:
I think the reason is clear at this point.

```suggestion
`/hpke_config` endpoint.
```

Reply:
Also changed `/hpke_config` to "when requesting the HPKE configuration from an Aggregator".